Formula/dafny.rb

```diff
@@ -51,9 +52,16 @@ def install
   test do
     (testpath/"test.dfy").write <<~EOS
       method Main() {
+        var i: nat;
+        assert i as int >= -1;
         print "hello, Dafny\\n";
       }
     EOS
-    system "#{bin}/dafny", testpath/"test.dfy"
+    assert_equal "Dafny 2.3.0.10506\n\nDafny program verifier finished with 1 verified, 0 errors\n",
```

Comment:
Is there any way you could write this test without hard-coding version information? This makes the formula difficult to maintain.

Reply:
Removed the version from the output using /nologo